tests/50federation/33room-get-missing-events.pl

```diff
@@ -413,3 +411,125 @@ sub sytest_user_and_room_fixture {
          Future->done;
       });
    };
+
+test "Outbound federation will ignore a missing event with bad JSON for room version 6",
+   requires => [ $main::OUTBOUND_CLIENT, $main::INBOUND_SERVER,
+                 local_user_and_room_fixtures(
+                    user_opts => { with_events => 1 },
```

Comment:
unless you specifically want a call to `/events` during registration, you don't need this. I think whereever you cargo-culted it from is wrong...